src/Types/PaymentType.res

```diff
@@ -122,7 +122,8 @@ type customerMethods = {
   paymentMethodIssuer: option<string>,
   card: customerCard,
 }
-type savedCardsLoadState = LoadingSavedCards | LoadedSavedCards(array<customerMethods>) | NoResult
+type savedCardsLoadState =
+  LoadingSavedCards | LoadedSavedCards(array<customerMethods>, bool) | NoResult(bool)
 
```

Comment:
why do you even need this bool? 
in both the variants .. cant this be solved by just by storing isGuestCustomer in a recoil and use it were ever needed

Reply:
then this is not even needed right you can just add this prop in item to obj mapper and get access to the value